docs/spicedb-dedicated/fgam.md

```diff
@@ -0,0 +1,141 @@
+
+The Policies page shows:
+
+- existing policies
+- navigate to the details page of a policy
+- delete a policy
+
+![Policies Page](/img/fgam/policies-page.png)
+
+## Advanced Roles with Conditions
+
+SpiceDB Dedicated Fine-Grained Access Management offers advanced permissions by letting users define dynamic conditions that need to be met at request time for the request to be authorized. This is achieved by augmenting fine-grained permissions with Google's CEL expression language.
+
+Each role permission provides an input text box to introduce an conditional expression, which is optional. When included, an API request will be authorized if:
+
+- the service account has been granted a permission
+- the expression of the permission is met
+
+![Advanced Role Conditions](/img/fgam/advanced-role-conditions.png)
+![advanced-role-conditions](https://user-images.githubusercontent.com/6774726/226452587-f8c3b9da-8e2d-4c13-861f-094b7719546e.png)
+
+SpiceDB Enterprise will expose the protobuf payload to the expression language, so the following variables will be available to be used in the expression:
+
+- `WriteRelationshipsRequest`
+- `ReadRelationshipsRequest`
+- `DeleteRelationshipsRequest`
+- `WriteSchemaRequest`
+- `ReadSchemaRequest`
+- `CheckPermissionRequest`
+- `LookupResourcesRequest`
+- `LookupSubjectsRequest`
+- `ExpandPermissionTreeRequest`
+- `WatchRequest`
+
+This is powerful because it let's you extend the logic to authorize an API request beyond "granted / not granted". Let's have a look at a few examples of what can be done:
+
+- Grants permissions to update only a specific type of resource on a write request: `WriteRelationshipsRequest.updates.all(x, x.relationship.resource.object_type == "resource")`
+- Grants permission to update only a specific type of subject on a write request: `WriteRelationshipsRequest.updates.all(x, x.relationship.subject.object.object_type == "user")`
+- Allows only write operations that use  `CREATE`: `"WriteRelationshipsRequest.updates.all(x, x.operation == authzed.api.v1.RelationshipUpdate.Operation.OPERATION_CREATE)"`
+- Allows reading only relationships of a specific type of resource type: `ReadRelationshipsRequest.relationship_filter.resource_type == "resource"`
+- Blocks writing the schema that contain specific strings: `!WriteSchemaRequest.schema.contains("blockchain")`
+- Only allows checking a specific permission: `CheckPermissionRequest.permission != "admin"`
+- Only allows looking up resources after a specific permission `LookupResourcesRequest.permission != "admin"`
+
+Any of the public API types will be avaliable to the expression language, so you can traverse any type and their fields using language operators. For more details on CEL's language definition, please refer to [CEL's language specification](https://github.com/google/cel-spec/blob/81e07d7cf76e7fc89b177bd0fdee8ba6d6604bf5/doc/langdef.md).
+
+## Example: assign read-only role to service account
```

Comment:
I was able to try and distill this guide into these bullets:

- Enable Fine-Grained Access Management when creating a new Permission System
- Navigate to the New Permission System's Access Management page
- Use the "Create Service Account" to create a Service Account with the desired name and description
- Navigate to the Roles Tab on the Access Management page
- Use the "Create Role" button to create a Role with the desired name and description
- Using the drop-down add the desired permissions leaving the conditional expressions empty
- Navigate to the Policies Tab on the Access Management page
- Use the "Create Policy" button to create a Policy that binds together the newly created Service Account and Role
- Now any Tokens created for the new Service Account will have read-only access to the APIs specified in the new Role


I'm not sure which format is better... trying to have the least text, but most valuable or having a fully featured guide.

I always try to write docs according to this talk: https://www.youtube.com/watch?v=cC67PzBgRYE
So I optimize for people that skim the page and aren't reading every detail.